autonomous/shoot_move_shoot.py

```diff
@@ -0,0 +1,235 @@
+import math
+
+from wpilib import controller
+from wpilib import geometry
+from wpilib import trajectory
+from wpilib.trajectory import constraint
+from magicbot import AutonomousStateMachine, state
+
+from components.chassis import Chassis
+from components.indexer import Indexer
+from components.shooter import Shooter
+from controllers.shooter import ShooterController
+
+
+def to_pose(x: float, y: float, heading: float) -> geometry.Pose2d:
```

Comment:
We should replace all calls to this to just call the Pose2d constructor, now that there is an overload that takes the angle directly rather than a Rotation2d object.